config/default.schema.json

```diff
@@ -1343,6 +1343,21 @@
 					"description": "Number of simultaneously synchronized students, teachers and classes"
 				}
 			}
+		},
+		"ADMIN_API_CLIENT": {
+			"type": "object",
+			"description": "Configuration of the schulcloud-server's admin API client.",
+			"properties": {
+				"BASE_URL": {
+					"type": "string",
+					"default": "http://localhost:4030",
```

Comment:
please move default. see example in lin
https://github.com/hpi-schul-cloud/schulcloud-server/main/config/default.schema.json#L477

Reply:
Okay, didn't know it can be done like that, thanks.